app/src/main/java/com/example/akademiaandroida/core/DI/NetworkModule.kt

```diff
@@ -0,0 +1,40 @@
+package com.example.akademiaandroida.core.DI
+
+import com.example.akademiaandroida.BuildConfig
+import com.example.akademiaandroida.features.data.RickAndMortyApi
+import com.example.akademiaandroida.utils.BASE_URL
+import okhttp3.Interceptor
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import org.koin.dsl.module
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+val networkModule = module {
+
+    single<Interceptor> {
+        HttpLoggingInterceptor()
+            .apply {
+                if (BuildConfig.DEBUG) HttpLoggingInterceptor.Level.BODY
```

Comment:
Tutaj brakuje przypisania wyniku tego bloku if else do zmiennej `level`. W opisie pod lekcją jest uwaga na ten temat, bo uciekło mi to podczas nagrywania.